contracts/TokenConverter/IAbstractTokenConverter.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { ResilientOracle } from "@venusprotocol/oracle/contracts/ResilientOracle.sol";
+
+/// @notice Interface for AbstractTokenConverter
+/// @custom:security-contact https://github.com/VenusProtocol/protocol-reserve#discussion
+interface IAbstractTokenConverter {
```

Comment:
Maybe add external views like `get[Updated]Amount{In,Out}` and `balanceOf` so that integrations don't have to import the contract?